src/server/ApiRoutes/Bib.js

```diff
@@ -230,6 +239,5 @@ export default {
   bibSearch,
   fetchBib,
   nyplApiClientCall,
-  addLocationUrls,
-  appendDimensionsToExtent,
```

Comment:
Why are we removing `appendDimensionsToExtent`?

Reply:
this is from an earlier version of this code. the function is now exported from its own file.